index.html

```diff
@@ -265,8 +279,95 @@ <h2 class="title homes-title">Homes guests loves</h2>
     </div>
   </div>
 </section>
+<section class="destinations">
+  <div class="container destinations-container">
+    <h1 class="title destinations-title">The best destinations</h1>
+    <form class="destinations-form" action="./" method="get">
+      <button class="button destinations-button regions">Regions</button>
+      <button class="button destinations-button cities">Cities</button>
+      <button class="button destinations-button places">Places</button>
+    </form>
```

Comment:
There is nothing to send and it no sense to wrap buttons into a form tag. It is not a form.